### ISSUE TITLE
LCD screen not working properly with v1.1 (shows the CLI but not the Blitz GUI)

### ISSUE THREAD

**Issue by Z-and** [raspiblitz/raspiblitz#478]
I'm in the process of setting up the RaspiBlitz (have previously setup the RaspiBolt) and the first thing I'm noticing is the LCD screen doesn't seem to match what's displayed on PuTTY. When I fire up the Pi the LCD screen will shoot a bunch of lines in the CLI (hard to read because of how small the screen is) and then just kinda gets stuck at one particular line. When I connect via PuTTY, the normal Blitz GUI comes up and I'm able to go through the configuration process but the LCD screen doesn't change at all. It stays stuck on the CLI with the same code that initialized during startup. Any ideas?

**Comment by fluidvoice** [comment]
Post what the LCD shows and also what shows on the HDMI monitor. Which exact LCD are you using?

**Comment by Z-and** [comment]
I've attached a couple photos. One of my PuTTY screen (I'm currently syncing the blockchain) and one of the LCD screen (hard to really capture). Also below is a link to the exact screen I bought off Amazon, looks to be the same one one the shopping list

https://www.amazon.com/gp/product/B075JFHLQD/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1

![PuTTY](https://user-images.githubusercontent.com/44408809/55294937-bd43fe00-53d5-11e9-83e1-f4cef6cc8e15.jpg)
![LCD Screen](https://user-images.githubusercontent.com/44408809/55294935-bb7a3a80-53d5-11e9-8ebf-7422275e39ac.JPG)

**Comment by openoms** [comment]
This is an HDMI screen, not the same to the ones in the shopping list.

**Comment by openoms** [comment]
Run this:
```
git clone https://github.com/goodtft/LCD-show.git
chmod -R 755 LCD-show
cd LCD-show/
sudo ./MPI3508-show
```
After executing the above command, the system will restart automatically, and display and touch will be OK.

Source: http://www.lcdwiki.com/3.5inch_HDMI_Display-B

If the letters will be still too small, you will need to resize the font which is described in this comment: https://github.com/rootzoll/raspiblitz/issues/244#issuecomment-476713706

**Comment by Z-and** [comment]
> This is an HDMI screen, not the same to the ones in the shopping list.

I see, will it still be compatible with this setup? There is an HDMI to HDMI part that connects the Pi to the screen, should I remove that and only connect with the pins?

**Comment by fluidvoice** [comment]
> > This is an HDMI screen, not the same to the ones in the shopping list.
> 
> I see, will it still be compatible with this setup? There is an HDMI to HDMI part that connects the Pi to the screen, should I remove that and only connect with the pins?

There is pins?  I thought this is HDMI connection only?  Don't remove the HDMI connector part.

**Comment by Z-and** [comment]
> > > This is an HDMI screen, not the same to the ones in the shopping list.
> > 
> > 
> > I see, will it still be compatible with this setup? There is an HDMI to HDMI part that connects the Pi to the screen, should I remove that and only connect with the pins?
> 
> There is pins? I thought this is HDMI connection only? Don't remove the HDMI connector part.

Yeah there are both pins and an HDMI connector.

**Comment by fluidvoice** [comment]
picture please

**Comment by Z-and** [comment]
It looks exactly like this except there is also an HDMI port, an Ext port, and what seems to be a headphone jack. I would take a photo of mine but I'm afraid to remove it since I'm in the middle of a sync.

![71M-759firL _SL1500_](https://user-images.githubusercontent.com/44408809/55295505-c8019180-53db-11e9-98d0-52fca6c8e186.jpg)

**Comment by Z-and** [comment]
> picture please

It looks exactly like this except there is also an HDMI port, an Ext port, and what seems to be a headphone jack. I would take a photo of mine but I'm afraid to remove it since I'm in the middle of a sync.

![71M-759firL _SL1500_](https://user-images.githubusercontent.com/44408809/55295522-f0898b80-53db-11e9-97f3-58b5aa8a6e5f.jpg)

**Comment by openoms** [comment]
@Z-and It probably looks like this: 
![Screenshot from 2019-04-01 00-16-09](https://user-images.githubusercontent.com/43343391/55296583-66f5b000-5413-11e9-8c3a-1b832ce4b14e.png)
@fluidvoice you can see the HDMI connector on his pic: 
![Screenshot from 2019-04-01 00-12-13](https://user-images.githubusercontent.com/43343391/55296543-e46cf080-5412-11e9-9b6b-16ace2e53133.png)

It will be compatible and you can connect it to the pins, but it only gets the power from there.
It can be also powered through the microUSB so you can have it separated from the Pi.
Once your sync is finished, just install it as described: https://github.com/rootzoll/raspiblitz/issues/478#issuecomment-478385232

**Comment by Z-and** [comment]
> @Z-and It probably looks like this:
> ![Screenshot from 2019-04-01 00-16-09](https://user-images.githubusercontent.com/43343391/55296583-66f5b000-5413-11e9-8c3a-1b832ce4b14e.png)
> @fluidvoice you can see the HDMI connector on his pic:
> ![Screenshot from 2019-04-01 00-12-13](https://user-images.githubusercontent.com/43343391/55296543-e46cf080-5412-11e9-9b6b-16ace2e53133.png)
> 
> It will be compatible and you can connect it to the pins, but it only gets the power from there.
> It can be also powered through the microUSB so you can have it separated from the Pi.
> Once your sync is finished, just install it as described: [#478 (comment)](https://github.com/rootzoll/raspiblitz/issues/478#issuecomment-478385232)

Thank you, that worked! Now I just need to figure out why my sync keeps failing. I have Bitcoin Core v0.17.1 fully synced up on a separate SSD connected to my computer. I updated the config file with txindex=1 and transferred all the necessary folders (blocks, chainstate, indexes) with WinSCP over to the Pi, but it keeps giving me the following message. 

![Capture 2](https://user-images.githubusercontent.com/44408809/55297054-e4a7c480-53ef-11e9-99f6-f74e6ab298d3.JPG)

**Comment by openoms** [comment]
I see you opened a new issue for the sync problems, so you can close this one. (I can`t)

Basicly you need to make sure that you have restarted your bitcoin client on the windows with txindex=1 and also closed it before starting the scp. https://github.com/rootzoll/raspiblitz/issues/479#issuecomment-478446366

**Comment by Z-and** [comment]
Resolved #478

**Comment by Z-and** [comment]
> Run this:
> 
> ```
> git clone https://github.com/goodtft/LCD-show.git
> chmod -R 755 LCD-show
> cd LCD-show/
> sudo ./MPI3508-show
> ```
> After executing the above command, the system will restart automatically, and display and touch will be OK.
> 
> Source: http://www.lcdwiki.com/3.5inch_HDMI_Display-B
> 
> If the letters will be still too small, you will need to resize the font which is described in this comment: [#244 (comment)](https://github.com/rootzoll/raspiblitz/issues/244#issuecomment-476713706)

Fixes #478

**Comment by Z-and** [comment]
> Run this:
> 
> ```
> git clone https://github.com/goodtft/LCD-show.git
> chmod -R 755 LCD-show
> cd LCD-show/
> sudo ./MPI3508-show
> ```
> After executing the above command, the system will restart automatically, and display and touch will be OK.
> 
> Source: http://www.lcdwiki.com/3.5inch_HDMI_Display-B
> 
> If the letters will be still too small, you will need to resize the font which is described in this comment: [#244 (comment)](https://github.com/rootzoll/raspiblitz/issues/244#issuecomment-476713706)

Fixes #478
